hypha/static_src/javascript/review-form-score.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+    "use strict";
+
+    // grab all the selectors
+    const selectors = Array.prototype.slice.call(
+        document.querySelectorAll("select")
+    );
+
+    if (selectors.length > 1) {
+        document.querySelector(".form--score-box").style.display = "block";
+        calculate_score();
+        selectors.forEach((selector) => {
+            selector.addEventListener("change", calculate_score);
+        });
+    }
+
+    function calculate_score() {
+        let score = 0;
+        selectors.forEach((selector) => {
+            const value = parseInt(selector.value);
+
+            if (!isNaN(value) && value !== 99) {
+                score += value;
+            }
+        });
+
+        // Update the text in .form--score-box with the current score
+        document.querySelector(".form--score-box").textContent =
+            "Score: " + score;
```

Comment:
Can we have the js only update the numeric value and not the text? Would we could to only add "Score" in the template so it can be translatable.

Reply:
An option to show total score or average score would be nice.

I think this also this feature should be off by default.

Reply:
The text is now translatable. I added an on-page button to switch between total and average (default total), though the style of it leaves something to be desired. I also added a new boolean system setting `REVIEW_SHOW_RUNNING_TOTAL` for it to be enabled. I suppose an alternative implementation would be to let the system setting have three settings (`None`, `Total`, and `Average`) but I hereby commend improvements to be an exercise for the reader.